aospy/test/test_roundtrip.py

```diff
@@ -150,33 +151,53 @@ def test_vert_int_sigma(self):
                                  dtype_out_time='av',
                                  dtype_out_vert='vert_int',
                                  dtype_in_vert='sigma',
-                                 **self.temp_test_params)
+                                 **self.three_d_test_params)
+        calc = Calc(calc_int)
+        calc.compute()
+
+    def test_sub_daily(self):
+        """Not tested for now in models other than idealized_moist.
+        and idealized_moist_rad."""
+        if not getattr(self, 'sub_daily_test_params', False):
```

Comment:
Can you you use the `unittest.skipif` decorator here instead?  That would be more idiomatic and consistent with the rest of the code.